submissions/OlexiyDobroskok/OOP_exercise/index.js

```diff
@@ -0,0 +1,146 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/OlexiyDobroskok/a-tiny-JS-world
+   Web app: https://olexiydobroskok.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor({
+    species = "No species",
+    name = "No name",
+    gender = "No gender",
+    saying = "Hello",
+    friends = [],
+  }) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  addFriend(person) {
+    this.friends.push(person);
+  }
+
+  introduceYourSelf() {
+    const template = `<strong>${this.saying}! My name is ${this.name}. Species: ${this.species}. Gender: ${this.gender}.</strong>`;
+    if (this.friends.length !== 0) {
+      const friendsName = this.friends.map((friend) => friend.name).join(", ");
+      return template + `<strong>Friends: ${friendsName}.</strong>`;
+    }
+    return template + `<strong>Friends: Looking for friends!</strong>`;
```

Comment:
You have something for any situation about friends (0 or non-zero number of friends).
`const friendsName = this.friends.length ? this.friends.map((friend) => friend.name).join(", "); : "Looking for friends"`
Now you can have a single `return ` statement. Altogether 2 lines of code for this method.
Feel free searching for ternary operators and/or seek for explanation in Students' chat.

Ternaries are helpful, you will use them often. Worth mastering them.

Reply:
@OleksiyRudenko Thanks. Looks cool) Need to look at ternary operators.